quic/s2n-quic-tls/src/server.rs

```diff
@@ -94,6 +94,10 @@ impl Default for Builder {
 }
 
 impl Builder {
+    pub fn mut_config(&mut self) -> &mut s2n_tls::config::Builder {
```

Comment:
nit: it's usually `config_mut` 

https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter